test/common/testlib.py

```diff
@@ -2092,6 +2094,34 @@ def enable_root_login(self):
         if not self.machine.ostree_image and self.file_exists(disallowed_conf):
             self.sed_file('/root/d', disallowed_conf)
 
+###########################
+
+
+
+class ExtendedMachineCase(unittest.TestCase):
```

Comment:
This does not actually extend the MachineCase class, it just inherits the default Python TestCase class. What we actually want to do is add these two functions below to the `MachineCase` class.